main.go

```diff
@@ -14,12 +14,14 @@ import (
 )
 
 type env struct {
-	GithubToken string `envconfig:"GITHUB_TOKEN"`
-	Owner       string `envconfig:"OWNER"`
-	Repo        string `envconfig:"REPO"`
-	PRNumber    int    `envconfig:"PR_NUMBER"`
-	Comment     string `envconfig:"COMMENT"`
-	MergeMethod string `envconfig:"MERGE_METHOD" default:"merge"`
+	GithubToken string   `envconfig:"GITHUB_TOKEN"`
+	Owner       string   `envconfig:"OWNER"`
+	Repo        string   `envconfig:"REPO"`
+	PRNumber    int      `envconfig:"PR_NUMBER"`
+	Comment     string   `envconfig:"COMMENT"`
+	MergeMethod string   `envconfig:"MERGE_METHOD" default:"merge"`
+	Mergers     []string `envconfig:"MERGERS"`
```

Comment:
```,```separated values are parsed to string slice.
https://github.com/kelseyhightower/envconfig#supported-struct-field-types